addons/l10n_br/models/account_journal.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models, api
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    l10n_br_invoice_serial = fields.Char(
+        'Series',
+        help='Brazil: Series number associated with this Journal. If more than one Series needs to be used, duplicate this Journal and assign the new Series to the duplicated Journal.'
```

Comment:
Should not that be automatically the case when you copy?  Should we put copy=False?